examples/cloud-run/tgi-deployment/README.md

```diff
@@ -234,22 +234,14 @@ The recommended approach is to use a Service Account (SA), as the access can be
       --region=$LOCATION
   ```
 
-- Update the Cloud Run service to use the Service Account:
```

Comment:
Removed this step because it's potentially confusing. It sets the service account as the service identity of the Cloud Run service, while this scenario shows how you can invoke the service using the service account credentials.